documentation/acceptence-tests/budget-management-tests.md

```diff
@@ -0,0 +1,18 @@
+# Budget Management Acceptance Tests
+
+## Acceptance test summary
+
+### create, view and modify budget
```

Comment:
Can you make sure all the titles for the acceptance tests are capitalized like this

```suggestion
# Budget Management Acceptance Tests

## Acceptance Test Summary

### Create, View and Modify Budget
```